tests/assets/mpii_json_dataset/dataset_with_numpy_files/mpii_annotations.json

```diff
@@ -0,0 +1 @@
+[{"dataset": "MPI","isValidation": 0.000,"img_paths": "000000001.jpg","img_width": 1280.000,"img_height": 720.000,"objpos": [594.000,257.000],"joint_self": [[620.000,394.000,1.000],[616.000,269.000,1.000],[573.000,185.000,1.000],[647.000,188.000,0.000],[661.000,221.000,1.000],[656.000,231.000,1.000],[610.000,187.000,0.000],[647.000,176.000,1.000],[637.020,189.818,1.000],[695.980,108.182,1.000],[606.000,217.000,1.000],[553.000,161.000,1.000],[601.000,167.000,1.000],[692.000,185.000,1.000],[693.000,240.000,1.000],[688.000,313.000,1.000]],"scale_provided": 3.021}, {"dataset": "MPI","isValidation": 0.000,"img_paths": "000000002.jpg","img_width": 1280.000,"img_height": 720.000,"objpos": [624.000,287.000],"joint_self": [[650.000,424.000,1.000],[646.000,309.000,1.000],[603.000,215.000,1.000],[677.000,218.000,1.000],[691.000,251.000,0.000],[686.000,261.000,1.000],[640.000,217.000,0.000],[677.000,216.000,1.000],[667.020,219.818,1.000],[725.980,138.182,1.000],[636.000,247.000,1.000],[583.000,191.000,1.000],[631.000,197.000,1.000],[722.000,215.000,1.000],[723.000,270.000,1.000],[718.000,343.000,1.000]],"scale_provided": 3.7}, {"dataset": "MPI","isValidation": 0.000,"img_paths": "000000003.jpg","img_width": 1280.000,"img_height": 720.000,"objpos": [564.000,227.000],"joint_self": [[590.000,364.000,1.000],[586.000,239.000,1.000],[533.000,155.000,1.000],[617.000,158.000,1.000],[631.000,191.000,1.000],[626.000,201.000,1.000],[580.000,157.000,1.000],[617.000,146.000,1.000],[607.020,159.818,1.000],[645.980,68.182,1.000],[576.000,187.000,1.000],[532.000,131.000,1.000],[571.000,137.000,1.000],[662.000,155.000,0.000],[663.000,210.000,1.000],[658.000,283.000,1.000]],"scale_provided": 3.2, "joint_others": [[490.000,264.000,1.000],[486.000,139.000,1.000],[433.000,55.000,1.000],[517.000,58.000,1.000],[531.000,91.000,1.000],[526.000,101.000,1.000],[480.000,57.000,1.000],[517.000,46.000,0.000],[507.020,59.818,1.000],[545.980,8.182,1.000],[476.000,87.000,1.000],[432.000,31.000,1.000],[471.000,37.000,1.000],[562.000,55.000,1.000],[563.000,110.000,1.000],[558.000,183.000,1.000], [690.000,464.000,1.000],[686.000,339.000,1.000],[633.000,255.000,1.000],[717.000,258.000,1.000],[731.000,291.000,1.000],[726.000,301.000,1.000],[680.000,257.000,1.000],[717.000,246.000,1.000],[707.020,259.818,1.000],[745.980,168.182,1.000],[676.000,287.000,1.000],[632.000,231.000,1.000],[671.000,237.000,1.000],[762.000,255.000,0.000],[763.000,310.000,1.000],[758.000,383.000,1.000]], "scale_provided_other": [2.65, 3.9], "objpos_other": [[464.000,127.000], [664.000,327.000]], "numOtherPeople": 2.000}]
```

Comment:
Why the values here contain 3 trailing zeros instead of 1? It doesn't seem to affect the value.